### PR TITLE
Establishing new Governance

### PR DESCRIPTION
**Pull request by sougou**
With the CNCF vote progressing favorably, we'd like to establish a governance policy, along with guiding principles and a code of conduct.

### CODE REVIEW

File: GUIDING_PRINCIPLES.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+*“To be the best performing, most scalable, and available NewSQL storage solution in the Cloud.”*
+
+Vitess can currently run on bare-metal. However, the trend is clear that applications are moving to the cloud. For the foreseeable future, we should continue to support the ability to run on bare-metal, because it will facilitate migration to the cloud.
+
+The Vitess architecture is generic enough to accommodate any other RDBMS in the future. However, Vitess currently runs on MySQL and has many features that provide compatibility with it. Vitess should remain focused on MySQL until all frictions are removed for those who wish to migrate from MySQL to Vitess.
+
+# Vision
+
+The vision of the Vitess project is to serve the software community by gaining popularity and adoption.
```
Comment (derekperkins) [inline GUIDING_PRINCIPLES.md]: This is super generic :)

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+## Community Code of Conduct
+
+Vitess follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
```
Comment (acharis) [inline CODE_OF_CONDUCT.md]: adopts

File: GOVERNANCE.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+* evangelising about the project (e.g. a link on a website and word-of-mouth awareness raising)
+* informing developers of strengths and weaknesses from a new user perspective
+* providing moral support (a ‘thank you’ goes a long way)
+* providing financial support (the software is open source, but its developers need to eat)
+
+Users who continue to engage with the project and its community will often become more and more involved. Such users may find themselves becoming contributors, as described in the next section.
+
+## Contributors
+
+Contributors will be added to the [Collaborators list](https://github.com/youtube/vitess/settings/collaboration).
```
Comment (acharis) [inline GOVERNANCE.md]: update link soon to vitessio ...

File: GUIDING_PRINCIPLES.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# Guiding Principles
+
+The Vitess project operates under the guiding principles described below. The PMC is responsible for upholding these principles and keeping this document updated with any changes.
+
+# Mission Statement
+
+*“To be the best performing, most scalable, and available NewSQL storage solution in the Cloud.”*
```
Comment (demmer) [inline GUIDING_PRINCIPLES.md]: This is slightly awkward phrasing... how about

> "To be the most performant, scalable, and available NewSQL storage system in the Cloud"

File: GOVERNANCE.md [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+* reporting bugs
+* identifying requirements
+* providing graphics and web design
+* programming
+* assisting with project infrastructure
+* writing documentation
+* fixing bugs
+* adding features
+
+Contributors engage with the project through the issue tracker,mailing list, the Slack channel, or by writing or editing documentation. They submit changes to the project itself via patches, which will be considered for inclusion in the project by existing committers (see next section). The Slack channel is the most appropriate place to ask for help when making that first contribution.
```
Comment (acharis) [inline GOVERNANCE.md]: tracker,mailing list -> tracker, mailing list [missing space]

File: GOVERNANCE.md [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+
+Nominees may decline their appointment as a committer. However, this is unusual, as the project does not expect any specific time or resource commitment from its community members. The intention behind the role of committer is to allow people to contribute to the project more easily, not to tie them in to the project in any formal way.
+
+It is important to recognise that commitership is a privilege, not a right. That privilege must be earned and once earned it can be removed by the PMC for conduct inconsistent with the [Guiding Principles](https://github.com/youtube/vitess/blob/master/GUIDING_PRINCIPLES.md) or if they drop below a level of commitment and engagement required to be a Committer, as determined by the PMC. The PMC also reserves the right to remove a person for any other reason inconsistent with the goals of Vitess.
```
Comment (acharis) [inline GOVERNANCE.md]: or for violating code of conduct?

File: GUIDING_PRINCIPLES.md (hunk 1 above)
Comment (sougou) [reply GUIDING_PRINCIPLES.md]: Intentional for now. We can iterate on this later.

File: GOVERNANCE.md (hunk 6 above)
Comment (sougou) [reply GOVERNANCE.md]: Code of conduct is included in the guiding principles.

**Pull request merged**